types/office-js-preview/index.d.ts

```diff
@@ -17165,6 +17256,42 @@ declare namespace Office {
          *                If the call fails, the `asyncResult.error` property will contain an error code with the reason for the failure.
          */
         getAllInternetHeadersAsync(callback?: (asyncResult: Office.AsyncResult<string>) => void): void;
+        /**
+         * Gets the Base64 encoding of a message.
+         *
+         * @remarks
+         * [Api set: Mailbox preview]
+         *
+         * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/understanding-outlook-add-in-permissions | Minimum permission level}**: **read item**
+         *
+         * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Message Read
+         *
+         * @param options - An object literal that contains one or more of the following properties:-
+         *        `asyncContext`: Developers can provide any object they wish to access in the callback function.
+         * @param callback - When the method completes, the function passed in the callback parameter is called with a single parameter, `asyncResult`, which is an
+         *          `Office.AsyncResult` object. The Base64 encoding of a message is returned in the `asyncResult.value` property. Any errors encountered are
+         *          returned in the `asyncResult.error` property.
+         *
+         * @beta
+         */
+        getAsFileAsync(options: Office.AsyncContextOptions, callback: (asyncResult: Office.AsyncResult<string>) => void): void;
+        /**
+         * Gets the Base64 encoding of a message.
```

Comment:
This should be something more like: 

Gets the current message in EML Format. The result is encoded in Base64.